_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.vue

```diff
@@ -819,3 +823,16 @@ export default {
   CampaignTypes,
 };
 </script>
+<style scoped>
+.blue-background {
+  border: 1px solid #174EEF;
+  background-color: #E3EAFD;
+  border-radius: 4px;
+  font-size: 0.75rem;
+  padding:  0.75rem;
+}
+
+.icon-color {
+  color: #174EEF
+}
+</style>
```

Comment:
I see we have a different way of displaying a notice alert. I'm not fond of this. We should update the whole app in a single time instead of having specific implementation everywhere. This is like a duplication of code.

Reply:
Should I go back to the previous version then ?

Reply:
Yes please, I checked with Sophie & Yassine